docker/api/network.py

```diff
@@ -33,6 +37,7 @@ def remove_network(self, net_id):
         url = self._url("/networks/{0}", net_id)
         res = self._delete(url)
         self._raise_for_status(res)
+        return True
```

Comment:
OK, I don't think either method should return anything, since the return value is always the same and therefore meaningless. @shin- - what was the reason for `remove_volume` returning `True`?